slider.js

```diff
@@ -0,0 +1,159 @@
+const DEFAULT_CONFIG = {
```

Comment:
It's better to use them as static properties of the class: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static